src/components/App.js

```diff
@@ -18,16 +19,22 @@ import {
 } from '../arrays/delegate-tasks';
 
 function App() {
+  const [professionPage, setProfessionPage] = React.useState(false);
+
   return (
     <div className="page">
-      <Header />
+      <Header professionPage={professionPage} setProfessionPage={setProfessionPage} />
       <Hero />
       <Lead />
+      <Requests setProfessionPage={setProfessionPage} />
       <Route path="/dummy-page">
```

Comment:
We can probably remove the dummy page now since we have the profession pages set up. 👍